src/widgets/attachments/attachments-gallery.css

```diff
@@ -0,0 +1,14 @@
+.documentsContainer {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+input[type="file"] {
```

Comment:
Not sure how this one gets scoped. My gut feeling is, that it is not. Hence please introduce a class to avoid clashes with other file inputs in the whole app.

Reply:
Thanks. Will fix.

Reply:
Is this better?

```typescript
.galleryHeader input[type="file"] {
  display: none;
}
```